src/zope/i18n/locales/inheritance.py

```diff
@@ -179,10 +179,9 @@ class InheritingDictionary(Inheritance, dict):
       >>> list(locale.data.items())
       [(1, 'eins'), (2, 'two'), (3, 'three')]
 
-    We currently fail to override ``values``, but instead inherited
-    data can be seen in the ``value`` method::
+    We also override ``values``::
 
-      >>> sorted(locale.data.value())
+      >>> sorted(locale.data.values())
       ['eins', 'three', 'two']
```

Comment:
Let's not sort, maybe -- the values should be returned in the same order as `keys()` and `items()`, and the tests for both of those do not call `sorted()`.

Reply:
Or maybe those others should call sorted? They are all the result of iterating across views of a dict, and dicts are inherently unordered (prior to 3.7, where the language spec has changed). The fact that the tests pass at all is an implementation artifact (hashes of ints are the int value, dicts iterate in hash order).